### DISCUSSION TITLE
Greetings Does anyone know when Gitpod will start allowing The Import of github copilot .vsix

### DISCUSSION THREAD

**Discussion by RitchZoeSquad** [community/community#16005] [Copilot Conversations]
Greetings Does anyone know when Gitpod will start allowing The Import of github copilot .vsix

**Comment by katorly** [comment]
Meeting [the same problem](https://github.com/github/feedback/discussions/10674), but they are not going to.  
Maybe you can ask them by opening a new issue [here](https://github.com/gitpod-io/gitpod/issues).

**Reply by petrtcoi** [reply]
The only way worked for me is to run local VS Code from Gitpod. Left top Burger Menu and than "Gitpod: open in VS Code"

**Comment by ArneCode** [comment]
You could install the Extension through the .vsix file which can be found [here](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot) (on the right side of the Page under Resources) through the gitpod.yml file [documentation](https://www.gitpod.io/docs/ides-and-editors/vscode-extensions) by providing an URL. You would have to host the file somewhere, though, but you can just save the file in a GitHub repo using the large file storage and use a link to that file. That said, I tried installing the extension this way and it actually appeared in the extensions tab but it told me that I don't have access to GitHub Copilot, even though it worked in other editors. My best guess would be that it either  Copilot recognizes that it is being run by Gitpod or (more likely) the authentication fails somewhere. You can see the Gitpod [here](https://gitpod.io/#https://github.com/ArneCode/CopilotTest)
